cmake-ide.el

```diff
@@ -944,6 +952,7 @@ the object file's name just above."
         (cmake-ide--message "Starting rdm server")
         (with-current-buffer buf (start-process "rdm" (current-buffer)
                                                 cmake-ide-rdm-executable
+						"-j 2" "-i 40" "-a 10"
```

Comment:
I just changed the number of jobs to 2 - you can use the previous configuration itself